pkg/addons/registry/registry.go

```diff
@@ -120,21 +121,21 @@ func (o *Registry) GenerateHelmConfig(k0sCfg *k0sv1beta1.ClusterConfig, onlyDefa
 		values = helmValues
 	}
 
-	// use a static cluster IP for the registry service based on the cluster CIDR range
-	serviceCIDR := k0sv1beta1.DefaultNetwork().ServiceCIDR
-	if k0sCfg.Spec != nil && k0sCfg.Spec.Network != nil && k0sCfg.Spec.Network.ServiceCIDR != "" {
-		serviceCIDR = k0sCfg.Spec.Network.ServiceCIDR
-	}
-
-	registryServiceIP, err := helpers.GetLowerBandIP(serviceCIDR, registryLowerBandIPIndex)
-	if err != nil {
-		return nil, nil, fmt.Errorf("failed to get cluster IP for registry service: %w", err)
-	}
-	values["service"] = map[string]interface{}{
-		"clusterIP": registryServiceIP.String(),
-	}
-
 	if !onlyDefaults {
```

Comment:
why remove the IPs from the default metadata? It just feels to me like this introduces the possibility of upgrade issues

Reply:
i guess i do not understand what onlyDefaults means then. what is not default about the tlsSecretName?

Reply:
"onlyDefaults" is what we pass when rendering the chart metadata for a release 

Apparently we don't include the tlsSecretName in that, but I don't understand why we also want to stop including the default service IP config

Reply:
If it's intentional that's fine but it didn't seem related to the major change to me